docs/internals/contributing/bugs-and-features.txt

```diff
@@ -88,16 +90,18 @@ Requesting features
 We're always trying to make Django better, and your feature requests are a key
 part of that. Here are some tips on how to make a request most effectively:
 
-* Make sure the feature actually requires changes in Django's core. If your
+* Be aware that your feature may not require changes in Django's core. If your
   idea can be developed as an independent application or module — for
   instance, you want to support another database engine — we'll probably
   suggest that you develop it independently. Then, if your project gathers
   sufficient community support, we may consider it for inclusion in Django.
 
-* First request the feature on the `Django Forum`_, not in the ticket tracker.
-  It'll get read more closely and reach a larger audience. This is even more
-  important for large-scale feature requests. We like to discuss any big
-  changes to Django's core before actually working on them.
+* First request the feature on the `new feature ideas tracker`_, not in the
+  ticket tracker. This is where the community and Steering Council evaluate
```

Comment:
We should add a link here for Steering Council to https://docs.djangoproject.com/en/dev/internals/organization/#steering-council